### PR TITLE
GEN-1106-glass damage UI

### PR DESCRIPTION
**Pull request by julia-andersson**
## [GEN-1106]

- Add glass damage UI 

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/Claims/Sources/Views/SubmitClaim/GlassDamage/SubmitClaimGlassDamageScreen.swift [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+                }
+
+                hSection {
+                    VStack(alignment: .leading, spacing: 8) {
+                        hText(L10n.submitClaimHowItWorksTitle)
+                        hText(L10n.submitClaimGlassDamageHowItWorksLabel)
+                            .foregroundColor(hTextColorNew.secondary)
+                    }
+                }
+                .padding(.top, 8)
```
Comment (sladan-hedvig) [inline Projects/Claims/Sources/Views/SubmitClaim/GlassDamage/SubmitClaimGlassDamageScreen.swift:36]: After we remove vertical spacing inside ClaimContractCard, we should switch this to 24 as it is in design

File: Projects/Claims/Sources/Views/SubmitClaim/GlassDamage/SubmitClaimGlassDamageScreen.swift [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+            hButton.MediumButtonFilled {
+                store.send(.dissmissNewClaimFlow)
+                DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
+                    store.send(.submitClaimOpenFreeTextChat)
+                }
+            } content: {
+                hText(L10n.CrossSell.Info.faqChatButton)
+            }
+            .frame(width: 133)
```
Comment (sladan-hedvig) [inline Projects/Claims/Sources/Views/SubmitClaim/GlassDamage/SubmitClaimGlassDamageScreen.swift:174]: Can we avoid setting up fixed width?

File: Projects/Claims/Sources/Views/SubmitClaim/GlassDamage/SubmitClaimGlassDamageScreen.swift [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+            .padding(.top, 24)
```
Comment (sladan-hedvig) [inline Projects/Claims/Sources/Views/SubmitClaim/GlassDamage/SubmitClaimGlassDamageScreen.swift:175]: We should not setup paddings here. Add that from where are calling this view

File: Projects/Contracts/Sources/View/CrossSellingItem.swift [hunk 4]
```diff
@@ -51,6 +51,7 @@ struct CrossSellingItem: View {
                     hText(L10n.crossSellGetPrice)
                 }
                 .hButtonConfigurationType(.primaryAlt)
+                .frame(width: 106)
```
Comment (sladan-hedvig) [inline Projects/Contracts/Sources/View/CrossSellingItem.swift:54]: Can we skip this setting of fixed width?

File: Projects/TravelCertificate/Sources/Views/WhoIsTravelingScreen.swift [hunk 5]
```diff
@@ -109,6 +109,7 @@ struct WhoIsTravelingScreen: View {
             }
         }
         .hButtonConfigurationType(.primaryAlt)
+        .frame(width: 148)
```
Comment (sladan-hedvig) [inline Projects/TravelCertificate/Sources/Views/WhoIsTravelingScreen.swift:112]: same

File: Projects/hCoreUI/Sources/hButton/hButton.swift [hunk 6]
```diff
@@ -13,10 +13,8 @@ struct LargeButtonModifier: ViewModifier {
 struct MediumButtonModifier: ViewModifier {
     func body(content: Content) -> some View {
         content
-            .padding(.horizontal, 16)
```
Comment (sladan-hedvig) [inline Projects/hCoreUI/Sources/hButton/hButton.swift:16]: We should not remove this. We need this to have same paddings everywhere for medium button

File: Projects/Contracts/Sources/View/CrossSellingItem.swift (hunk 4 above)
Comment (julia-andersson) [reply Projects/Contracts/Sources/View/CrossSellingItem.swift:54]: how?

**Review by sladan-hedvig — APPROVED** [review]

**Pull request merged**